main/lsp/lsp_helpers.cc

```diff
@@ -226,10 +226,10 @@ string prettyTypeForConstant(const core::GlobalState &gs, core::SymbolRef consta
     }
 
     if (constant.isTypeAlias(gs)) {
-        // By wrapping the type in `MetaType`, it displays as `<Type: Foo>` rather than `Foo`.
-        result = core::make_type<core::MetaType>(result);
+        return fmt::format("T.type_alias {{{}}}", result.show(gs));
```

Comment:
If I understand correctly, the thing we substitute for `{}` is the rhs of the alias statement, right? If that's correct, maybe `T.type_alias for {}` would read better.

Reply:
The thing that users actually write is `T.type_alias {Integer}` not `T.type_alias for Integer`, and I was hoping to change it so that it would read just like users typed it (makes it easy in case they want to e.g. select and copy it).